src/discord/commands/book.ts

```diff
@@ -174,7 +174,7 @@ export default {
         let selectMenuDate = new MessageActionRow().addComponents(new MessageSelectMenu().setCustomId('dateSelectMenu').setPlaceholder('Select Date of Room Booking').addOptions(getDateOptions()));
         let menuSelectedDate: string;
 
-        const message = (await interaction.reply({ embeds: [embed], components: [selectMenuDate], fetchReply: true })) as Message;
+        const message = (await interaction.reply({ content: `<@!${interaction.user.id}>`, embeds: [embed], components: [selectMenuDate], fetchReply: true })) as Message;
```

Comment:
```suggestion
        const message = (await interaction.reply({ content: `${interaction.user}`, embeds: [embed], components: [selectMenuDate], fetchReply: true })) as Message;
```

Nitpick but you can just do this - putting it in a `${}` block calls the `toString` method on `interaction.user` which formats it as a mention automatically.